src/Sulu/Bundle/HttpCacheBundle/Cache/SuluHttpCache.php

```diff
@@ -55,4 +60,12 @@ public function fetch(Request $request, $catch = false)
     {
         return parent::fetch($request, $catch);
     }
+
+    protected function createStore()
+    {
+        return new Psr6Store([
+            'cache_directory' => $this->kernel->getWebsiteCacheDir(),
```

Comment:
Is using the website kernel really necessary here? Due to sulu/sulu-minimal#150 the cache should be only active in the website context anyway. And apart from that, this would probably lead to strange behavior in case we start caching some stuff in the admin context as well.